go/teams/teams.go

```diff
@@ -850,3 +874,35 @@ func LoadTeamPlusApplicationKeys(ctx context.Context, g *libkb.GlobalContext, id
 	}
 	return team.ExportToTeamPlusApplicationKeys(ctx, keybase1.Time(0), application)
 }
+
+func (t *Team) loadAllTransitiveSubteams(ctx context.Context) ([]*Team, error) {
+	subteams := []*Team{}
+	for subteamID := range t.Data.Chain.SubteamLog {
+		// Load each subteam...
+		subteam, err := Load(ctx, t.G(), keybase1.LoadTeamArg{
+			ID:          subteamID,
+			NeedAdmin:   true,
+			ForceRepoll: true,
+		})
+		if err != nil {
+			return nil, err
+		}
+
+		// Force loading the key manager.
+		// TODO: Should this be the default, so that we don't need to do it here?
+		_, err = subteam.SharedSecret(ctx)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
I think it's possible to hit an error if `Team.Data.PerTeamKeySeeds` is empty for some reason, or if somehow it gets inconsistent with `TeamSigChainState.inner.PerTeamKeys`? I'll be conservative and leave it like this, at least until the next time we run into it.